src/templates/base.html

```diff
@@ -30,207 +30,210 @@
+                <div class="ui inverted link list">
+                    <a href="#" class="item">Banana Pre-Order</a>
+                    <a href="#" class="item">DNA FAQ</a>
+                    <a href="#" class="item">How To Access</a>
+                    <a href="#" class="item">Favorite X-Men</a>
                 </div>
             </div>
+            <div class="seven wide column">
+                <h4 class="ui inverted header">Footer Header</h4>
+                <p>Extra space for a call to action inside the footer that could help re-engage users.</p>
+            </div>
         </div>
     </div>
-</div>
 
-{# Admin only user switching #}
-<user_switch></user_switch>
-
-<!-- Vendor -->
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js" integrity="sha256-7/yoZS3548fXSRXqc/xYzjsmuW3sFKzuvOCHd06Pmps=" crossorigin="anonymous"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/riot/3.13.2/riot+compiler.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.js"></script>
-<script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
-<script src="{% static "js/calendar.min.js" %}"></script>
-<script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/dompurify/1.0.10/purify.min.js"></script>
-
-<!-- Ours -->
-<script src="{% static "js/ours/client.js" %}"></script>
-<script src="{% static "js/ours/init.js" %}"></script>
-<script src="{% static "js/ours/riotjs_mixins.js" %}"></script>
-<script src="{% static "js/ours/utils.js" %}"></script>
-<script src="{% static "js/jquery.form.serializer.js" %}"></script>
-<script src="{% static "js/Chart.bundle.js" %}"></script>
-<script src="{% static "js/reconnecting-websocket.min.js" %}"></script>
-<script src="{% static "generated/riot.js" %}"></script>
-
-<script>
-    {# From `common_settings` context processor #}
-    STORAGE_TYPE = "{{ STORAGE_TYPE }}"
-    CODALAB.state.user = {
-        id: {{ request.user.id }},
-        name: '{{ request.user.name }}',
-        username: '{{ request.user.username }}',
-        email: '{{ request.user.email }}',
-        bio: '{{ request.user.bio }}',
-        // converts python bools to js bools
-        is_staff: '{{ request.user.is_staff }}' === "True",
-        is_superuser: '{{ request.user.is_superuser }}' === "True",
-        has_competition_admin_privileges: competition => {
-            let user = CODALAB.state.user
-            if (user.is_superuser || user.is_staff) {
-                return true
+
+    {# Admin only user switching #}
+    <user_switch></user_switch>
+
+    <!-- Vendor -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Doesn't look like this list got updated entirely.
Think it should be this:
```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.min.js" integrity="sha256-7/yoZS3548fXSRXqc/xYzjsmuW3sFKzuvOCHd06Pmps=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/riot/3.13.2/riot+compiler.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.js"></script>
<script src="https://cdn.jsdelivr.net/particles.js/2.0.0/particles.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
<script src="{% static "js/calendar.min.js" %}"></script>
<script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/dompurify/1.0.10/purify.min.js"></script>
```
Plus probably luxon and any other vender library you added

Reply:
Fixed in latest push

Reply:
Looks like toastr and calendar.min.js are in there twice. 
And we aren't using simplemde.min.js anymore, or select2. 
So those duplicates/adipose imports can get cut. My mistake for not catching that w/ my first comment.

Reply:
Removed duplicates in latest push